components/brave_shields/content/browser/brave_shields_util.cc

```diff
@@ -156,8 +156,7 @@ std::string ControlTypeToString(ControlType type) {
     case ControlType::DEFAULT:
       return "default";
     default:
-      NOTREACHED_IN_MIGRATION();
-      return "invalid";
+      NOTREACHED_NORETURN();
```

Comment:
Are these all the possible options? If it is, we should remove the `default:` entry, and have `NOTREACHED_NORETURN` outside the switch at the end of the function, because that way we get a build error if a new key is added to the switch.